server/sonar-server/src/main/java/org/sonar/server/component/ws/AppAction.java

```diff
@@ -181,12 +205,6 @@ private ComponentDto retrieveRootIfNotCurrentComponent(ComponentDto componentDto
   }
 
   @CheckForNull
-  private static String formatCoverageMeasure(Map<String, MeasureDto> measuresByMetricKey) {
-    MeasureDto coverage = measuresByMetricKey.get(CoreMetrics.COVERAGE_KEY);
-    return formatMeasure(coverage, CoreMetrics.COVERAGE);
-  }
-
-  @CheckForNull
   private static String formatMeasure(Map<String, MeasureDto> measuresByMetricKey, Metric metric) {
     MeasureDto measure = measuresByMetricKey.get(metric.getKey());
     return formatMeasure(measure, metric);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "getRootUuid"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)